macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -160,6 +160,7 @@
 
     {% if not elementary.has_temp_table_support() %}
         {% do adapter.drop_relation(temp_relation) %}
+        {% do adapter.clean_up_table(temp_relation) %}
```

Comment:
@bebbo203 - this fails in Databricks since it doesn't have this column.
I'm guessing you should use `fully_drop_relation` here as well?